app/views/analysis/useranalysis.scala.html

```diff
@@ -0,0 +1,82 @@
+@()(implicit request: Request[_], messages: Messages, flash: Flash)
+
+@main("Analysis") {
```

Comment:
Change title to User Knolx Analysis.
And change the Knolx Analysis page title to "Knolx Analysis".